Yosemite/Yosemite/Model/Mocks/MockObjectGraph.swift

```diff
@@ -81,6 +107,94 @@ extension MockObjectGraph {
     }
 }
 
+// MARK: SiteSetting Creation Helper
+extension MockObjectGraph {
+    static func createSiteSetting(siteID: Int64 = 1,
+                                  settingID: String,
+                                  label: String,
+                                  settingDescription: String,
+                                  value: String,
+                                  settingGroupKey: String) -> SiteSetting {
+        SiteSetting(siteID: siteID,
```

Comment:
I was thinking about using Fakeable+Copiable here, but realized that this isn't in a test target 😬

Reply:
Yeah that's why we have such large changes for this file 😅